build/commands/lib/config.js

```diff
@@ -210,6 +211,7 @@ Config.prototype.buildArgs = function () {
     enable_cdm_host_verification: this.enableCDMHostVerification(),
     skip_signing: !this.shouldSign(),
     chrome_pgo_phase: this.chromePgoPhase,
+    use_blink_v8_binding_new_idl_interface: this.useBlinkV8BindingNewIdlInterface,
```

Comment:
can you add a comment here similar to the commit comment?

Reply:
Addressed by 4b5c88cac9f3170aa0b3691e413788325f7c57db